src/freedv_api.c

```diff
@@ -1372,6 +1372,28 @@ int freedv_get_bits_per_modem_frame       (struct freedv *f) {return f->bits_per
 int freedv_get_rx_status                  (struct freedv *f) {return f->rx_status;}
 void freedv_get_fsk_S_and_N               (struct freedv *f, float *S, float *N) { *S = f->fsk_S[0]; *N = f->fsk_N[0]; }
 
+
+/*---------------------------------------------------------------------------*\
+
+  FUNCTIONS...: freedv_set_fmin_fmax
+  AUTHOR......: Simon Lang - DJ2LS
+  DATE CREATED: 18 feb 2022
+
+  Set fmin / fmax for ofdm modem
+
+\*---------------------------------------------------------------------------*/
+int freedv_set_fmin_fmax(struct freedv *freedv, float val_fmin, float val_fmax) {
+
+    if (is_ofdm_data_mode(freedv)) {
+        freedv->ofdm->fmin = val_fmin;
```

Comment:
Should we also be checking if this is "burst mode"?  This API function just adjusts the acquisition range of the burst data modems (used for raw data mode).

Reply:
so you are talking about `is_ofdm__mode(*f)` instead of `is_ofdm_data_mode(*f)` ?
This would add additional support for modes 2020, 700D, 700E - right?

Reply:
Burst mode is defined by `ofdm->data_mode` being set to "burst" (see line 1310 in ofdm.c).

Reply:
thanks for help, @tmiw !  I pushed a commit with this changes and I hope this is what you and @drowe67 where talking about.

Reply:
Yep, but the check should be if `strcmp(...) == 0` (it'll return non-zero if not equal).

Reply:
oh, sorry! I expected some returned "bool" value. Should have read C docs first!. However, I updated the corresponding line, recompiled and ran all ctests with success.